github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -45,13 +45,23 @@ presubmits:
           value: "kind-1.19-sriov"
         - name: "KUBEVIRT_NUM_NODES"
           value: "3"
+        - name: "RUN_KUBEVIRT_CONFORMANCE"
+          value: "true"
+        - name: "SONOBUOY_EXTRA_ARGS"
+          value: "--plugin-env kubevirt-conformance.E2E_FOCUS=SRIOV"
         command:
         - "/usr/local/bin/runner.sh"
         - "/bin/bash"
         - "-c"
         - >
-            trap "echo teardown && make cluster-down" EXIT ERR SIGINT SIGTERM &&
-            make cluster-up
+          nightly_build_base_url="https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt"
+          latest=$(curl -L "${nightly_build_base_url}/latest")
+          commit=$(curl -L "${nightly_build_base_url}/${latest}/commit")
+          container_tag="${latest}_${commit:0:9}"
+          SONOBUOY_EXTRA_ARGS="--plugin-env kubevirt-conformance.CONTAINER_TAG=${container_tag} ${SONOBUOY_EXTRA_ARGS}"
+          trap "echo teardown && make cluster-down" EXIT ERR SIGINT SIGTERM &&
```

Comment:
EXIT SIGINT are enough please

note that if make cluster-down is failing, it will override the return code fwiw

Reply:
Done